scalableminds/webknossos
# Update style of login, signup views

I updated the style of the login, signup, password reset, and change password pages as suggested by Albane. Mostly I just added a colorful, flashy background with some neurons. 

The included SVG is not the smallest images. About 280kb gzipped. I tried to "optimize" it with [SVGOmg](https://jakearchibald.github.io/svgomg/) but the savings were neglectable.

Tested with Chrome, Safari, Firefox.

<img width="1622" alt="image" src="https://user-images.githubusercontent.com/1105056/218506835-535a5f81-c246-49f1-a373-2ee9e5a5a066.png">

<img width="1617" alt="image" src="https://user-images.githubusercontent.com/1105056/218506755-42b70c15-cb4c-4279-afc1-a1077480d623.png">


### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open all the 4 pages and check styling
- Use dev tools to try responsive styling
- Try dark mode

### Issues:
- None

------
(Please delete unneeded items, merge only when none are left open)
